requirements.in

```diff
@@ -0,0 +1,13 @@
+langchain_community
+langchain
+langchain_core
+langchain_ibm
+flask
+streamlit
+tiktoken
+unstructured
+markdown
+# For GPU ones
+#faiss-gpu
```

Comment:
I don't have a GPU :'(
Feel free @justjais to comment this out and uncomment yours for main.